support/sier-codec/tests/integrated.rs

```diff
@@ -164,3 +164,55 @@ fn struct_() {
 
     assert_eq!(obj["foo"].as_u8(), Some(42));
 }
+
+const JSON_STRUCT_DEF: &'static str = r#"
+struct Foo {
+    bar :u64;
+    baz :string;
+    qux :List<u64>;
+    corge :Corge;
+}
+"#;
+
+const JSON_STRUCT_DEF2: &'static str = r#"
```

Comment:
Do we need this as a separate string? I thought we supported multiple structs per file.